File: Python/Program/Application.py
from SoftWare import SoftWare

class Application(SoftWare):
    def __init__(self, name="", version="", category="", license_=""):
        super().__init__(name, version)
        self.category = category
        self.license = license_

    def set_category(self, category):
        self.category = category

    def set_license(self, license_):
        self.license = license_

    def get_category(self):
        return self.category

    def get_license(self):
        return self.license

File: Java/Program/Application.java
public class Application extends SoftWare {
    private String category;
    private String license;

    public Application() {}

    public Application(String name, String version, String category, String license) {
        super(name, version);
        this.category = category;
        this.license = license;
    }

    public void setCategory(String category) { this.category = category; }
    public void setLicense(String license) { this.license = license; }

    public String getCategory() { return category; }
    public String getLicense() { return license; }
}
